portfolio/src/main/webapp/personal.html

```diff
@@ -5,7 +5,7 @@
     <title>Personal Tomi</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
-    <script src="https://maps.googleapis.com/maps/api/js?key=$MAP_API_KEY"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyD6c0fosAzG1jaRst6mOAJdGuZex3lbOWo"></script>
```

Comment:
It is definitely not a desirable practice to have the API key submitted, but I guess as long as the github repo is private or the API key you are using is not paid for with money, it doesn't matter.